Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -795,6 +795,16 @@ public void ReadXmlPrototype(XmlReader readerParent)
                     break;
             }
         }
+
+        if (orderActions.ContainsKey("Uninstall"))
+        {
+            InventoryCommon asInventory = new InventoryCommon();
+            asInventory.type = Type;
+            asInventory.maxStackSize = 1;
+            asInventory.basePrice = 0f;
+            asInventory.category = "crated_furniture";
+            PrototypeManager.Inventory.Add(asInventory);
+        }
```

Comment:
what does this do? If it's needed can't it be placed in UnInstall.Initialize() ?